esx_service/utils/auth_data.py

```diff
@@ -664,7 +663,10 @@ def new_db(self):
         err = self.__create_default_tenant()
         if err:
             return err
-        err = self.__create_default_privileges()
+        err = self.__create_all_ds_privileges_for_default_tenant()
+        if err:
+            return err
+        err = self.__create_vm_ds_privileges_for_default_tenant()
```

Comment:
Ok, but can the user run the same command again, after new_db() returns failure and would it cause any issue? In config_init() there is a check to see if a DB already exists and ignores calling new_db.

Possibly add a test to simulate error return for __create_vm_ds_privileges_for_default_tenant()  and check behavior when user retries "config init"